core/app/models/spree/line_item.rb

```diff
@@ -88,12 +87,11 @@ def pre_tax_amount
     alias display_total display_amount
     deprecate display_total: :display_amount, deprecator: Spree::Deprecation
 
-    # Sets price and currency from a `Spree::Money` object
+    # Sets price from a `Spree::Money` object
     #
-    # @param [Spree::Money] money - the money object to obtain price and currency from
+    # @param [Spree::Money] money - the money object to obtain price from
     def money_price=(money)
```

Comment:
done, but [assigning nil to line item price when money is nil](https://github.com/solidusio/solidus/pull/1507/commits/73660fa3f4249b745bbd39fb7caa177cd4e18c12).